src/Mindee.Cli/Commands/PredictCommand.cs

```diff
@@ -146,7 +146,7 @@ private async Task<int> EnqueueAndParseAsync(InvocationContext context, ParseOpt
                     new LocalInputSource(options.Path),
                     new PredictOptions(allWords: AllWords, fullText: FullText),
                     null,
-                    new AsyncPollingOptions(maxRetries: 60));
+                    new AsyncPollingOptions(maxRetries: 80));
```

Comment:
ideally should be a CLI arg, but for now just remove it since the default is now higher